fast_plotter/plotting.py

```diff
@@ -492,6 +558,32 @@ def convert_intervals(vals):
         vals = vals.mid
     return vals
 
+def parse_dtype_args(other_dtype_args):
```

Comment:
This can probably be moved to `__main__` and could also probably be achieved more elegantly